History.md

```diff
@@ -1,3 +1,7 @@
+# UNRELEASED
+
+* Fix Cucumber steps to use `have_same_file_content_as` matcher
```

Comment:
Could you add ` (#572)` at the end of this line? That'd harmonize it with the other entries.

Reply:
Sure! Now I get my number 😄 Amended into the last commit 9146d2d